countdown.js

```diff
@@ -1,5 +1,15 @@
-function countdown(seconds){
-  // ...
+// This seemed like the simplest way to do this without combining a loop with a timer or getting into recursion. 
+//Since Javascript Garden suggests that setTimeout is used over setInterval, I'm curios about outher implementations.
```

Comment:
> Javascript Garden suggests that setTimeout is used over setInterval

Ah, interesting. Can you bring this up in class on Thursday?

> I'm curios about other implementations

Many to choose from!

https://github.com/advanced-js/countdown/pulls?utf8=%E2%9C%93&q=is%3Apr

Here's a recursive one:

https://github.com/advanced-js/countdown/pull/36/files